easybuild/easyconfigs/c/CheckM/CheckM-1.0.18-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,45 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'CheckM'
+version = '1.0.18'
+
+homepage = 'https://github.com/Ecogenomics/CheckM'
+description = """CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,
+ single cells, or metagenomes."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+builddependencies = [('binutils', '2.31.1')]
+
+dependencies = [
```

Comment:
It looks like this should depend on SciPy-bundle  (random script i looked at https://github.com/Ecogenomics/CheckM/blob/master/scripts/distributionDeltaTetraDiff.py#L32 )

Reply:
Indeed, @Darkless012 please add `SciPy-bundle` as a dependency (`numpy` was included with `Python` itself in older toolchains)